YOLayoutExample/YOLayoutExample/SwiftView/CircleLayoutView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  CircleLayoutView.swift
+//  YOLayoutExample
+//
+//  Created by John Boiles on 1/9/15.
+//  Copyright (c) 2015 YOLayout. All rights reserved.
+//
+
+//! A view lays out its subviews in a circle.. in Swift!
+class CircleLayoutView: YOView {
+    var imageViews = NSMutableArray()
+
+    override func viewInit() {
+        self.backgroundColor = UIColor(white: 0.96, alpha: 1)
+
+        // Create a set of subviews
+        for i in 0...7 {
+            let imageView = UIImageView(image: UIImage(named: "information.png"))
+            self.addSubview(imageView)
+        }
+
+        self.layout = YOLayout(layoutBlock: { [unowned self] (layout : YOLayoutProtocol!, size) -> CGSize in
+            var angle : CGFloat = 0;
+            let center = CGPointMake(size.width / 2, size.height / 2)
+            let subviewSize = CGSizeMake(44, 44)
+
+            // TODO: Switch this to `let` when Swift 1.2 is officially released
+            var radius : CGFloat;
+            if (size.width < size.height) {
+                radius = size.width / 2 - subviewSize.width / 2
+            } else {
+                radius = size.height / 2 - subviewSize.width / 2
+            }
+
+            // Lay out subviews in a circle around center
+            for subview in self.subviews {
+                // Trigonometry, bitches
```

Comment:
maybe change this to fuck yeah or something else... just to be insanely hypersensitive about women in tech issues :)